<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profit</class>
 <widget class="QMainWindow" name="Profit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(255, 255, 127);
border-radius: 12px;
	border-color: rgb(0, 170, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-color: rgb(170, 0, 127);
	color: rgb(170, 0, 127);
	background-color: rgb(255, 170, 255);
border-radius:14px;
}</string>
    </property>
    <property name="text">
     <string>Прибутки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 0);
	border-color: rgb(255, 255, 0);
	background-color: rgb(255, 0, 0);
border-radius: 12px;
}
QPushButton:hover:!pressed { 
background-color: white;
border-color: black;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>55</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(255, 170, 0);
	background-color: rgb(255, 255, 0);
	border-color: rgb(255, 170, 0);
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Спонсори</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_res">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

border-radius: 10px;
	background-color: rgb(0, 255, 0);
	border-color: rgb(255, 170, 0);
	color: rgb(255, 170, 0);
}
QPushButton:hover:!pressed { 
background-color: white;
border-color: black;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Обрахувати</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>681</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_samsung">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Samsung</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_adidas">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Adidas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_nike">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Nike</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_puma">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Puma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_spotify">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Spotify</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_chevrolet">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Chevrolet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_toyota">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Toyota</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_flyemirates">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Fly Emirates</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_parimatch">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Pari Match</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_huawei">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Huawei</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_xiaomi">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Xiaomi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_newbalance">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>New Balance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_mcdonnalds">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>McDonnalds</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_cocacola">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Coca Cola</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_visa">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
color: rgb(170, 0, 0);
	border-color: rgb(170, 0, 0);
	background-color: rgb(170, 255, 127);
border-radius: 10px;
height: 35px;
}</string>
         </property>
         <property name="text">
          <string>Visa</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>260</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>Проданих футболок:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>Ціна футболки:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>Кількість проведених матчів:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>Кількість місць на стадіоні:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>Ціна білету на матч:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>255</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_numfotballs">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 0, 0);
border-radius: 8px;
height: 30px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pricefootballs">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 0, 0);
border-radius: 8px;
height: 30px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_matches">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 0, 0);
border-radius: 8px;
height: 30px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_numplaces">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 0, 0);
border-radius: 8px;
height: 30px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_priceticket">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border-color: rgb(0, 0, 0);
border-radius: 8px;
height: 30px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>231</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color: rgb(255, 170, 127);
	color: rgb(170, 0, 127);
	border-color: rgb(170, 0, 127);
border-radius: 8px;
}</string>
       </property>
       <property name="text">
        <string>Результат:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_result">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 0, 0);
	background-color: rgb(255, 255, 255);
	border-color: rgb(0, 0, 127);
border-radius: 6px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
